acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/privacy/PrivacyNode.java

```diff
@@ -105,7 +104,7 @@ public PrivacyNode(final PrivacyNodeConfiguration privacyConfiguration, final Ve
             besuConfig.getExtraCLIOptions(),
             Collections.emptyList(),
             besuConfig.getPrivacyParameters(),
-            Optional.empty());
+            List.of());
```

Comment:
(optional) `emptyList`

Reply:
They are interchangeable - https://stackoverflow.com/a/46619739/8020
I prefer List.of() to train myself to use List.of(...) for fixed content lists.

Reply:
Sounds good :+1: